lib/presentation/molecules/tip_resource_molecule.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'package:infinite_horizons/presentation/atoms/separator_atom.dart';
+import 'package:infinite_horizons/presentation/core/global_variables.dart';
+import 'package:infinite_horizons/presentation/molecules/molecules.dart';
+
+ValueNotifier<bool> isFullScreen = ValueNotifier(false);
```

Comment:
There is no need to use ValueNotifier, just use a regular variable inside of the statefulwidget and use a callback function